gst/nnstreamer/nnstreamer_plugin_api_impl.c

```diff
@@ -1662,9 +1661,17 @@ gst_tensor_buffer_append_memory (GstBuffer * buffer, GstMemory * memory,
     return FALSE;
   }
 
-  if (!gst_tensor_info_validate (info)) {
-    nns_loge ("Failed to get tensor info (invalid input info).");
-    return FALSE;
+  if (gst_tensor_meta_info_parse_memory (&meta, memory)) {
```

Comment:
Don't need to check the size of memory?

Reply:
Agreed, I will make another PR for this.

Reply:
Added new PR #4148 to check memory size before parsing tensor-meta.